packages/ag-charts-community/src/chart/agChartV2.ts

```diff
@@ -390,7 +392,7 @@ class AgChartsInternal {
 }
 
 function applyChartOptions(chart: Chart, processedOptions: ProcessedOptions, userOptions: AgChartOptions): void {
-    const completeOptions = jsonMerge([chart.processedOptions ?? {}, processedOptions], noDataCloneMergeOptions);
+    const completeOptions = mergeDefaults(processedOptions, chart.processedOptions);
```

Comment:
Is the precedence here correct? The new options state (`processedOptions`) should take precedence over the previously determined options state (`chart.processedOptions`)?